src/clientversion.h

```diff
@@ -44,8 +44,10 @@ static const int CLIENT_VERSION =
 extern const std::string CLIENT_NAME;
 extern const std::string CLIENT_BUILD;
 
+std::string FormatVersion(int nVersion);
 std::string FormatFullVersion();
```

Comment:
We can use this opportunity to change this into something more meaningful. The current terminology introduces a lot of confusion - Version vs Subversion vs FullVersion. (Full version lacks clarity on what exactly is full and opens up for mistakes in the future). 

`FormatFullVersion` appears to be version with suffix (like branch names). How about renaming this to  `FormatVersionAndSuffix` instead?

Reply:
Okay